.cargo/config.toml

```diff
@@ -41,4 +41,4 @@ linker = "lld-link.exe"
 MACOSX_DEPLOYMENT_TARGET = "10.10"
 
 [build]
-rustdocflags = ["--document-private-items"]
```

Comment:
I think this change was unintentional.

Reply:
No config.toml was not formatted, so I needed to format it for CI to work.